lib/collection/set.js

```diff
@@ -26,9 +27,6 @@ export class SetColl extends Collection {
   }
 
   _addWithoutObserver(item) {
```

Comment:
```suggestion
  _addWithoutObserver(item) {
    if (!item) {
      throw new Error("null objects are not allowed");
    }
```

Reply:
... so this suggestion is even worse than the original code.

Reply:
Yes, forgot about `0`. Could you fix that, please?

Reply:
I've adapted my suggestion. Does that make more sense?

Reply:
Yes, thanks,